### DISCUSSION TITLE
getting error while sending notification - { "error": { "code": 100, "fbtrace_id": "ANczUs30-94RWJItbqH1R81", "message": "(#100) The parameter template['language']['code'] is required.", "type": "OAuthException" } }

### DISCUSSION THREAD

**Discussion by rishira1** [shridarpatil/frappe_whatsapp#38] [Q&A]
Can anybody please help it is urgent. Checked template at facebook. able to send message using curl with same template

**Comment by chechani** [comment]
the language code as per facebook and erpnext are different...go to whatsapp template doctype and fetch templates from meta
